test/e2e/pages-dir/next-dynamic/with-static-import/pages/dynamic-import.tsx

```diff
@@ -0,0 +1,9 @@
+import dynamic from 'next/dynamic'
```

Comment:
does it only work with dynamic?

sometimes we lazy load a hook or a non react component - in those cases we don't use dynamic - could we add a test for `import()` without dynamic?

Reply:
Added, thank you!